spec.md

```diff
@@ -567,9 +567,22 @@ the resource it creates.
 | service_id*  | string  | ID of the service from the catalog.  |
 | plan_id*  | string  | ID of the plan from the catalog.  |
 | app_guid  | string  | Deprecated in favor of <code>bind\_resource.app\_guid</code>. GUID of an application associated with the binding to be created.  |
-| bind_resource  | JSON object  | A JSON object that contains data for platform resources associated with the binding to be created. Current valid values include <code>app\_guid</code> for [credentials](#types-of-binding) and <code>route</code> for [route services](#route-services).  |
+| bind_resource  | JSON object  | A JSON object that contains data for platform resources associated with the binding to be created. See [Bind Resource Object](#bind-resource-object) for more information. |
 | parameters | JSON object  |  Configuration options for the service binding. An opaque object, controller treats this as a blob. Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |   |
 
+##### Bind Resource Object
+
+| Request Field | Type | Description |
+| --- | --- | --- |
+| app_guid | string | GUID of an application associated with the binding. For [credentials](#types-of-bindings) bindings. |
+| route | string | URL of the application to be intermediated. For [route services](#route-services) bindings. |
+
+Note: the `app_guid` is OPTIONAL because based on the platform's use of the
```

Comment:
Suggestion: change this verbiage:

> because based on the platform's use of the service binding there might not be an application known at this time.

to 'because the consuming application may not be known when the binding is created'.